ocfweb/docs/templates/docs/partials/account.md

```diff
@@ -0,0 +1,35 @@
+Many think of the Open Computing Facility as an on-campus resource for free printing. While this is true, and printing is certainly our most-demanded service (we print hundreds of thousands of pages per year), there are plenty of other great services provided by the OCF, which come free with your account!
+
+Note: the following information applies to individual accounts, not group accounts. To request an account for your student group, or for information on student group web hosting, check out [our group account page](https://www.ocf.berkeley.edu/docs/services/vhost/).
+
+## Free Computing
+
+The OCF was founded to provide all members of the Cal campus community with computing resources. Anytime we're open, all members of the community are welcome to use the computers for any reason – not just to print. All of the computers in the OCF run Linux and have a variety of useful applications pre-installed.
+* Google Chrome, Firefox, and Brave are all installed, allowing you to choose your preferred web browser.
+* The LibreOffice suite is installed as an alternative to Microsoft Office. It is mostly cross-compatible with Office files, meaning one can import or export .docx/.pptx/.xlsx files. You can also use Google Docs from a web browser.
+* GIMP (GNU Image Manipulation Program) is provided as an alternative to Adobe Photoshop, and Inkscape replaces Adobe Illustrator. For animation and 3D modeling, we provide access to Blender.
+* Rather than iTunes/QuickTime or Windows Media Player, you can access music or video files using VLC.
+* Pre-installed IDEs include Atom and Visual Studio Code.
+* The default PDF utility is Document Viewer, which can be used to view, print, and edit PDFs.
+* Steam is installed, if you would like to play video games.
+Most of these applications can be found on the desktop, and all applications can be accessed from the menu in the top-left corner of the screen, analogous to the Start menu in Windows. We also offer three computers with dedicated scanners, and one with a CD/DVD reader.
+
+## Persistent File Storage
+
+When you log out of one of our desktop computers, any files you saved to that computer will be erased. This is because most files saved to these computers are immediately printed, so there is no need to take up disk space holding onto them. However, some users may wish to have access to their files every time they log in to an OCF desktop. As part of your OCF account, you gain access to 15 GB of free file storage on OCF servers. This file storage is accessible by clicking on the "OCF File Storage" icon on any desktop computer or by SSHing into our account server (tsunami.ocf.berkeley.edu, and if you don't know what SSH means, read [our guide to the shell](LINK-GOES-HERE)).
```

Comment:
Perhaps "SSHing into our account server" could simply be a link to the SSH instructions page.

Reply:
updated